pkg/systemd/services/service.jsx

```diff
@@ -72,26 +73,28 @@ export class Service extends React.Component {
         const url = "/system/logs/#/?prio=debug&service=" + cur_unit_id;
 
         return (
-            <Page groupProps={{ sticky: 'top' }}
-                  isBreadcrumbGrouped
-                  id="service-details"
-                  breadcrumb={
-                      <Breadcrumb>
-                          <BreadcrumbItem to={"#" + cockpit.location.href.replace(cockpit.location.path[0], '')}>{_("Services")}</BreadcrumbItem>
-                          <BreadcrumbItem isActive>
-                              {this.props.unit.Id}
-                          </BreadcrumbItem>
-                      </Breadcrumb>}>
-                <PageSection className="ct-pagesection-mobile">
-                    <Gallery hasGutter>
-                        <GalleryItem id="service-details-unit">{serviceDetails}</GalleryItem>
-                        {((this.props.unit.LoadState === "loaded" || this.props.unit.LoadState === "masked") && this.props.owner == "system") &&
-                        <GalleryItem id="service-details-logs">
-                            <LogsPanel title={_("Service logs")} match={match} emptyMessage={_("No log entries")} max={10} goto_url={url} search_options={{ prio: "debug", service: cur_unit_id }} />
-                        </GalleryItem>}
-                    </Gallery>
-                </PageSection>
-            </Page>
+            <WithDialogs>
+                <Page groupProps={{ sticky: 'top' }}
```

Comment:
This should be in the previous commit, not the "only reset units.." one.

Reply:
Ouch, yes.

Reply:
I have moved this into its own PR: #17327  It has nothing to do with this change here anymore.